resources/assets/js/features/wizard-trip/reducer.js

```diff
@@ -0,0 +1,51 @@
+import * as actions from './actionTypes';
+import {INIT, STEP_ONE, STEP_TWO, STEP_THREE} from 'app/services/WizardTripService';
+
+const initialState = {
+    step: INIT,
```

Comment:
all props that describes selected data should be separated from rest props, put them in data or pendingTrip or tripToSave property